### PR TITLE
Adding Starboard to the addons

### PR DESCRIPTION
**Pull request by AnaisUrlichs**
Adding the starboard operator to the microk8s addons

https://github.com/aquasecurity/starboard

### Thank you for making MicroK8s better

Please reference the issue this PR is fixing, or provide a description of the problem addressed.

*Also verify you have:*
* [x] Read the [contributions](https://github.com/ubuntu/microk8s/blob/master/CONTRIBUTING.md) page.
* [x] Submitted the [CLA form](https://ubuntu.com/legal/contributors/agreement), if you are a first time contributor.

### CODE REVIEW

**Comment by neoaggelos** [comment]
This looks very interesting!

**Review by neoaggelos — APPROVED** [review]
LGTM, thanks @AnaisUrlichs!

**Comment by neoaggelos** [comment]
Can you please sign the CLA and fix the formatting in the Python tests (using `black`)?

File: tests/validators.py [hunk 1]
```diff
@@ -605,6 +605,13 @@ def validate_openebs():
     assert "my-data" in output
     kubectl("delete -f {}".format(manifest))
 
+def validate_starboard():
+    """
+    Validate Starboard
+    """
+    wait_for_pod_state(
+        "", "starboard", "running", label="app.kubernetes.io/component=server"
```
Comment (neoaggelos) [inline tests/validators.py:613]: Also, judging from the starboard operator that is applied on the cluster, I think this should be

```suggestion
        "", "starboard-system", "running", label="app.kubernetes.io/instance=starboard-operator"
```

**Comment by ktsakalozos** [comment]
Hi @AnaisUrlichs have you signed the CLA agreement? The test is failing but I see you have checked the box on your PR. I wonder if this is an issue with the CLA checker.

**Comment by AnaisUrlichs** [comment]
I think it's an issue with my GH account/email

**Pull request merged**